py_src/ipyelk/layouting/elkjs.py

```diff
@@ -26,7 +26,8 @@ def message(self) -> Dict:
 
 
 class ElkJS(DOMWidget):
-    """Jupyterlab widget for interacting with ELK diagrams"""
+    """Jupyterlab widget for calling [elkjs](https://github.com/kieler/elkjs)
```

Comment:
this is using markdown style links, but in `ElkDiagram` we were using RST style?  I prefer MD but should we keep it the same?

Reply:
good catch. i prefer MD too but for the current sphinx setup the links need to be RST. I am sure there is a way to pipe them through a separate transform but for now I am aiming for RST for links in docstrings.